cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PolicyReplicationsCompletedListener.java

```diff
@@ -0,0 +1,41 @@
+package com.cloudant.sync.replication;
```

Comment:
Yes, it should, annoyingly the IDE seems to miss that step when extracting classes instead of creating them from new - must be a tick box somewhere I need to find!